lib/ClangImporter/ImporterImpl.h

```diff
@@ -1862,6 +1862,12 @@ static inline Type applyToFunctionType(
   return type;
 }
 
+inline bool unavailableOnSwift(clang::Decl *decl) {
```

Comment:
There is already an `Impl.isUnavailableInSwift()` helper; would that not suffice for this? If not, what is the difference in logic between the two helpers, and should this one have a name that reflects the difference more clearly?

Also, nit: `unavailableInSwift` (not `On`) would better match terminology elsewhere in the compiler.

Reply:
Yep, that works. Thanks.